vignettes/VectorControl_Bednets.Rmd

```diff
@@ -0,0 +1,129 @@
+  dn0 = matrix(c(.533, .533), nrow = 2, ncol = 3), # Matrix of death probabilities for each mosquito species over time
+  rn = matrix(c(.56, .56), nrow = 2, ncol = 3), # Matrix of repelling probabilities for each mosquito species over time
+  rnm = matrix(c(.24, .24), nrow = 2, ncol = 3), # Matrix of minimum repelling probabilities for each mosquito species over time
+  gamman = rep(2.64 * 365, 2) # Vector of bed net half-lives for each distribution timestep
+)
+
+output <- run_simulation(timesteps = sim_length, parameters = bednetparams)
+```
+
+### Plot prevalence
+```{r, fig.align="center", fig.height=5, fig.width=7}
+plot_prev()
+```
+  
+## Comparing input coverage and population bed net usage 
+
+It is important to understand the difference between the input `coverages` argument of `set_bednets()` and the resulting population bed net usage over time in the model. When we set coverages in the above example to 0.8, we are telling the model to distribute bed nets to 80% of the population at year one, and to 50% of the population at year 2. However, the level of average bed net usage is not necessarily equal to 80% or 50%. Between these time points, bed net use will slowly decline over time (in reality this decline stems from things like disuse, holes in nets, lost nets, etc.).
+  
+The average population bed net usage will be influenced by:   
+
+* The size and frequency of distributions specified in `set_bednets()`  
+* The assumed net retention half life (`gamman`)
+* Correlations in the recipients of nets between rounds  
+  
+The output from `malariasimulation::run_simulation()` has a variable `n_use_net` that shows the number of people using bed nets at any given timestep. We can visualise the proportion of the population using bed nets over time to understand how bed net usage changes. 
+
+```{r, fig.align="center", fig.height=5, fig.width=7}
+output$prop_use_net <- output$n_use_net / human_population
+
+plot(x = output$timestep, y = output$prop_use_net, type = "l", 
+     col = cols[3], lwd = 2.5, ylim = c(0,1),
+     xlab = "Timestep (days)", ylab = "Proportion of population using bed nets",
+     xaxs = "i", yaxs = "i", bty = "l")
+grid(lty = 2, col = "grey80", lwd = 1)
+axis(side = 1, lty = 1, col = "black", pos = 0); axis(side = 2, lty = 1, col = "black")
+```
+  
+## Using the `netz` package to estimate coverage inputs needed to achieve target population usage
+
+The [`netz` package](https://mrc-ide.github.io/netz/index.html) is a useful tool to help set up bed nets in `malariasimulation`. `malariasimulation` takes as input the proportion of the population who were distributed a bed net at specific time points, but available data for a specific region or country do not always have this information. `netz` has functionality to estimate modelled population usage over time in the simulation for a given set of input coverages. It can also help to fit the input coverages to usage data.
```

Comment:
"at specific time points, but net distribution or use data are not always available for a specific region or country"
"for a given set of input distributions. It can also ... "